examples/swarm/image/docker/Makefile

```diff
@@ -0,0 +1,20 @@
+ifndef PGVERSION
```

Comment:
Isn't this Makefile the same as the one in `examples/kubernetes/image`?

Reply:
same reply as above